gbasis/evals/density.py

```diff
@@ -195,6 +202,25 @@ def evaluate_deriv_density(
 ):
     r"""Return the derivative of density of the given transformed basis set at the given points.
 
+    .. math::
+        \begin{align}
+            &\frac{\partial^{L_x + L_y + L_z}}{\partial x^{L_x} \partial y^{L_y} \partial z^{L_z}}
+            \rho(\mathbf{r}_n)\\
+            &=
+            \sum_{l_x=0}^{L_x} \sum_{l_y=0}^{L_y} \sum_{l_z=0}^{L_z}
+            \binom{L_x}{l_x} \binom{L_y}{l_y} \binom{L_z}{l_z}
+            \sum_{ij} \gamma_{ij}
+            \frac{\partial^{l_x + l_y + l_z} \rho(\mathbf{r}_n)}{\partial x^{l_x} \partial y^{l_y} \partial z^{l_z}}
+            \frac{
+                \partial^{L_x + L_y + L_z - l_x - l_y - l_z} \rho(\mathbf{r}_n)
+            }{
+                \partial x^{L_x - l_x} \partial y^{L_y - l_y} \partial z^{L_z - l_z}
+            }
+        \end{align}
+
+    where :math:`L_x, L_y, L_z` are the orders of the derivative relative to the :math:`x, y, \text{and} z` components,
+    respectively.
```

Comment:
1. Change $\mathbf{r}_n$  to $\mathbf{r}$.
2. The two lines seem to be numbered, like two equations.